spot-ml/src/main/scala/org/apache/spot/lda/SpotLDAWrapperSchema.scala

```diff
@@ -19,6 +19,9 @@ package org.apache.spot.lda
 
 import org.apache.spark.sql.types.{LongType, StringType, StructField}
 
+/**
+  * Schemas and column names used in SpotLDAWrapper and *SuspiciousConnectsModel.
```

Comment:
Oh.  Since there's just three right now, you might as well list them.  As it is, it looks like a typo.